crm_multicompany_reporting_currency/models/crm_lead.py

```diff
@@ -37,6 +38,15 @@ def _get_multicompany_reporting_currency_id(self):
         readonly=True,
     )
 
+    # technical field to be used in sum_field options on kanban
+    current_company_currency = fields.Many2one(
```

Comment:
`Many2one` fields should conventionally end up with `_id`
```suggestion
    current_company_currency_id = fields.Many2one(
```